arch/x86/core/intel64/locore.S

```diff
@@ -562,6 +573,21 @@ except: /*
 
 	iretq
 
+#ifdef CONFIG_X86_KPTI
+EXCEPT      ( 0, 7); EXCEPT      ( 1, 7); EXCEPT       (2, 6); EXCEPT      ( 3, 7)
```

Comment:
can we macro the second argument for clarity, EXC_STACK, NMI_STACK, etc

Reply:
@andrewboie if I use macro names like EXC_STACK, NMI_STACK they are not expanded, this code (https://github.com/finikorg/zephyr/blob/up-clean/arch/x86/core/intel64/locore.S#L434) produces
`__x86_tss64_t_istEXC_STACK_OFFSET` instead of `__x86_tss64_t_ist7_OFFSET`